docker/images/n8n/README.md

```diff
@@ -1,111 +1,108 @@
 
-## Start with tunnel
+To save your work between container restarts, it also mounts a docker volume, `n8n_data`. The workflow data gets saved in an SQLite database in the user folder (`/home/node/.n8n`). This folder also contains important data like the webhook URL and the encryption key used for securing credentials.
 
-> **WARNING**: This is only meant for local development and testing. Should not be used in production!
+If this data can't be found at startup n8n automatically creates a new key and any existing credentials can no longer be decrypted.
 
-To be able to use webhooks which all triggers of external services like Github
-rely on n8n has to be reachable from the web. To make that easy n8n has a
-special tunnel service (uses this code: [https://github.com/n8n-io/localtunnel](https://github.com/n8n-io/localtunnel)) which redirects requests from our servers to your local
-n8n instance.
+## Start n8n with tunnel
+
+> **WARNING**: This is only meant for local development and testing and should **NOT** be used in production!
+
+n8n must be reachable from the internet to make use of webhooks - essential for triggering workflows from external web-based services such as GitHub. To make this easier, n8n has a special tunnel service which redirects requests from our servers to your local n8n instance. You can inspect the code running this service here: [https://github.com/n8n-io/localtunnel](https://github.com/n8n-io/localtunnel)
 
 To use it simply start n8n with `--tunnel`
 
 ```bash
+docker volume create n8n_data
+
 docker run -it --rm \
  --name n8n \
  -p 5678:5678 \
- -v ~/.n8n:/home/node/.n8n \
+ -v n8n_data:/home/node/.n8n \
  docker.n8n.io/n8nio/n8n \
  start --tunnel
 ```
 
-## Persist data
-
-The workflow data gets by default saved in an SQLite database in the user
-folder (`/home/node/.n8n`). That folder also additionally contains the
-settings like webhook URL and encryption key.
-Note that the folder needs to be writable by user with UID/GID 1000.
-
-```bash
-docker run -it --rm \
- --name n8n \
- -p 5678:5678 \
- -v ~/.n8n:/home/node/.n8n \
- docker.n8n.io/n8nio/n8n
-```
-
-### Start with other Database
+## Use with PostgreSQL
 
-By default n8n uses SQLite to save credentials, past executions and workflows.
-n8n however also supports PostgresDB.
+By default, n8n uses SQLite to save credentials, past executions and workflows. However, n8n also supports using PostgreSQL.
 
-It is important to still persist the data in the `/home/node/.n8n` folder. The reason
-is that it contains n8n user data. That is the name of the webhook
-(in case) the n8n tunnel gets used and even more important the encryption key
-for the credentials. If none gets found n8n creates automatically one on
-startup. In case credentials are already saved with a different encryption key
-it can not be used anymore as encrypting it is not possible anymore.
+Note that even when using a different database, it is still important to
```

Comment:
It might be worth using the same convention here that we use for the warning above (quoted, with a bolded label). I could see people missing this on a skim.